packages/pharos/src/components/icon/pharos-icon.ts

```diff
@@ -30,13 +30,38 @@ export class PharosIcon extends PharosElement {
   /**
    * A description of what the icon represents
    * @attr description
+   * @deprecated Please use a11yLabel instead.
    */
   @property({ type: String, reflect: true })
   public description = '';
 
+  /**
+   * Indicates the svg title to apply to the icon.
```

Comment:
I think it might be worthwhile to call out that the title is the SVG accessible name.
```suggestion
"Indicates the title to apply as the accessible name of the icon"
```